src/system/ThemeModule/Resources/config/services.yml

```diff
@@ -84,10 +95,11 @@ services:
           $combine: '%zikula_asset_manager.combine%'
 
     Zikula\ThemeModule\Bridge\Twig\EventEnabledTwigEngine:
-        decorates: templating.engine.twig
+        decorates: twig
         arguments:
-          $parser: '@templating.name_parser'
-          $locator: '@templating.locator'
+          $loader: '@twig.loader'
```

Comment:
@Guite given your previous comments, perhpahs this is the problem. Should `$loader` be something else?

Reply:
No, I think the missing extensions are not related to any type of template loader.